omod/src/main/compass/sass/patientHeader.scss

```diff
@@ -74,36 +74,6 @@ $green: #51A351;
         top: -15px;
       }
     }
-
-    .colored-message {
-      @include horizontal-list;
-
-      color: white;
-      padding: 2px 4px;
-      border-radius: 1px;
-      margin-top: 4px;
-      font-size: 0.8em;
-    }
-
-    .bordered-colored-message {
-      @extend .colored-message;
-
```

Comment:
No, I did not. Sorry the commit may have included some reformatting ? 
Only thing I've changed in this file is adding the
.secondLineFragments { 
  clear: both;
}